ui/dashboards/src/context/DashboardProvider/dashboard-provider-api.ts

```diff
@@ -24,18 +24,16 @@ export function useEditMode() {
  * Returns actions that can be performed on the current dashboard.
  */
 export function useDashboardActions() {
-  const { addPanelGroup, reset, save } = useDashboardStore(({ addPanelGroup, reset, save }) => ({
-    addPanelGroup,
-    reset,
-    save,
-  }));
-  const addPanel = useDashboardStore((store) => store.addPanel);
+  const save = useDashboardStore((store) => store.save);
```

Comment:
how is calling `useDashboardStore` each time different than getting multiple states at once?

Reply:
Good question: the only difference is avoiding the more expensive shallow comparison check. Zustand basically says "use whichever style you want" in the docs, so I've just been doing the "cheaper" one.

Reply:
Edit: I just realized that when you created `useDashboardStore` you added the shallow comparison check automatically 😂 🤦‍♂️  

Thinking maybe we should just remove it, but if you like the object-style selectors better, also totally cool with me to do that instead. 🤷‍♂️